cns/types/types.go

```diff
@@ -499,6 +499,62 @@ func init() {
 	types.Add("CnsQueryVolumeInfoResult", reflect.TypeOf((*CnsQueryVolumeInfoResult)(nil)).Elem())
 }
 
+type CnsRelocateVolumeRequestType struct {
+	This          types.ManagedObjectReference `xml:"_this"`
+	RelocateSpecs []BaseCnsVolumeRelocateSpec  `xml:"relocateSpecs"`
```

Comment:
Add `typeattr` here, so the `XSIType` field isn't needed.

Reply:
Thanks for the tip. Didn't know that. :)